Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/SpeechInputHandler.cs

```diff
@@ -108,6 +138,16 @@ void IMixedRealitySpeechHandler.OnSpeechKeywordRecognized(SpeechEventData eventD
             {
                 keywordResponse.Invoke();
                 eventData.Use();
+
+                // OS will display the tooltip for the "Select" keyword. 
+                if ((speechConfirmationTooltipPrefabInstance != null) && (eventData.Command.Keyword.ToLower() != "select"))
+                {
+                    // Update the text label with recognized keyword
+                    speechConfirmationTooltipPrefabInstance.GetComponentInChildren<TextMeshPro>().text = eventData.Command.Keyword;
```

Comment:
Hm...it's a little unusual that this component requires textmeshpro (using TMPro) just so it can set the text here. Also, what if you wanted to use a different text label, not text mesh pro?

What if you had a different approach, something like this:

```
class TooltipConfirmation : Monobehavior {
    public virtual void SetText(string s);
    public virtual void Confirmed();
}

class HoloLens2Tooltip : BaseTooltip {
   public void SetText(string s) {
       textMeshPro.text = s;
   }
    public virtual void Confirmed() {
         kickOffAnimation();
     }
```

This will remove tooltip's dependency on textmeshpro, and allow people to use tooltips without needing textmeshpro (they can make their own).

In SpeechInputHandler, reference tooltip like this:

```
private TooltipConfirmation speechConfirmationTooltipPrefab = null;
```

Reply:
I see, making it more generic so that developers can put any object with TextMesh/UIText/TextMesh Pro.
Or making it more specific to the provided SpeechConfirmationTooltipPrefab.

Reply:
Added SpeechConfirmationTooltip which inherits the ToolTip class and provide generic interface for the SetText and animation trigger. Updated SpeechInputHandler to call these.